.github/workflows/ci.yml

```diff
@@ -148,6 +148,42 @@ jobs:
     - name: test
       run: python check.py --binaryen-bin=out/bin
 
+  # Build with gcc 6.3 and run tests on Alpine Linux (inside chroot).
+  # Note: Alpine uses musl libc.
+  # Keep in sync with build_release.yml
+  build-alpine:
+    name: alpine
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x'
+    - uses: actions/checkout@v1
+    - name: start docker
+      run: |
+        docker run -w /src -dit --name alpine -v $PWD:/src node:lts-alpine
+        echo 'docker exec alpine "$@";' > ./alpine.sh
```

Comment:
Is this the only thing in alpine.sh? Why not write the "docker exec..." command out everywhere instead? That's slightly more duplication, but also seems more straightforward. My first thought when I saw a reference to alpine.sh was that it was a file I would be able to find in the Binaryen source tree.

Reply:
I didn't write this code, I'm just copying it from build_release.yml .. if we want to make that change its can be done as a followup I think,